### PR TITLE
GR: implement Soultender

### PR DESCRIPTION
**Pull request by strib**

### CODE REVIEW

File: server/game/cards/07-GR/Soultender.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    setupCardAbilities(ability) {
+        this.play({
+            reap: true,
+            gameAction: ability.actions.removeAmber((context) => ({
+                all: true,
+                target: context.game.creaturesInPlay.filter((c) => c.hasTrait('specter'))
+            })),
+            then: {
+                alwaysTriggers: true,
+                gameAction: ability.actions.ward((context) => ({
```
Comment (romanogit) [inline server/game/cards/07-GR/Soultender.js:15]: You need to fully heal the specter creatures as well.

**Review by romanogit — CHANGES_REQUESTED** [review]

File: server/game/cards/07-GR/Soultender.js (hunk 1 above)
Comment (strib) [reply server/game/cards/07-GR/Soultender.js]: Oops sloppy, good catch.  Fixed!

File: server/game/cards/07-GR/Soultender.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            gameAction: ability.actions.removeAmber((context) => ({
+                all: true,
+                target: context.game.creaturesInPlay.filter((c) => c.hasTrait('specter'))
+            })),
+            then: {
+                alwaysTriggers: true,
+                gameAction: [
+                    ability.actions.heal((context) => ({
+                        target: context.game.creaturesInPlay.filter((c) => c.hasTrait('specter')),
+                        fully: 3
```
Comment (romanogit) [inline server/game/cards/07-GR/Soultender.js:18]: well, it will work, but should be `fully: true`, right?

Comment (strib) [reply server/game/cards/07-GR/Soultender.js]: Haha, sorry, yes, thanks for fixing.

**Pull request merged**